lib/components/ArtistScreen/artist_screen_content.dart

```diff
@@ -112,11 +115,12 @@ class _ArtistScreenContentState extends State<ArtistScreenContent> {
           var albums = snapshot.data?.elementAtOrNull(1) ?? [];
           var songsByPlaycount = allSongs.then((songs) {
             var sortedsongs = List<BaseItemDto>.from(songs ?? []);
-            sortedsongs.sort(
-              (a, b) =>
-                  b.userData?.playCount.compareTo(a.userData?.playCount ?? 0) ??
-                  0,
-            );
+            sortedsongs.sort((a, b) {
+              int compareCount = (b.userData?.playCount ?? 0)
+                  .compareTo(a.userData?.playCount ?? 0);
+              if (compareCount != 0) return compareCount;
+              return (a.name ?? "").compareTo(b.name ?? "");
```

Comment:
There should be a `nameForSorting` attribute on the `BaseItemDto` to emulate Jellyfin's sorting behavior (e.g. excluding articles for albums, ignoring case, etc.), please compare that instead!